blocks/empty-section/empty-section.js

```diff
@@ -0,0 +1,3 @@
+/**
```

Comment:
I don't understand the empty-section block, why do we need it?

Reply:
I need an Empty section to add feedback. Perhaps I could be called it just feedback, but sometimes an empty section is added without knowledge and an error is shown at the console warning us that that section was not able to load section.js, or something like this. So to avoid that lib-franklin error, I add this feature

Reply:
several things here:
- I would not call it section as it actually is a block and a section is a different construct in Franklin
- if we do not have explicit requirements that the authors need to position this area freely on the page, independent from the from I would just create the needed area/divs with Javascript, that way the authors would not need to know that they always need to create the additional block when adding a form

Reply:
I removed the empty-section block because now is not needed anymore.